lib/CleantalkSP/SpbctWP/Firewall/UploadChecker.php

```diff
@@ -341,17 +344,51 @@ public static function runCheckForWordpressModules(
      */
     private function checkUploadedArchive($archive_path)
     {
+        global $wp_filesystem;
+
         if ( !empty($archive_path) && is_string($archive_path)) {
-            //todo perform unzipping logic there and adopt the result
-            //then use checkFileContent() to check file
-            $unzipped_files = array();
-            foreach ($unzipped_files as $path) {
-                $result = $this->checkFileContent($path);
-                if (false !== $result) {
+            require_once ABSPATH . 'wp-admin/includes/file.php';
```

Comment:
This line above is redundand?

Reply:
You are right, fixed https://github.com/CleanTalk/security-malware-firewall/pull/285/commits/a08f17f289eaccee6d0fc1e050846e24b5d2afae